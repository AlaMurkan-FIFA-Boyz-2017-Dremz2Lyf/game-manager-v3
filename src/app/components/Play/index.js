import React from 'react';
import PropTypes from 'prop-types';
import { Container, Segment, Header, Message } from 'semantic-ui-react';
import Game from '../Game';
import games from '../Game/tests/fixtures';

const Play = ({ tournament }) => (
  <Container>
    <Segment.Group>
      <Segment clearing>
        <Header icon="soccer" content="Tournament Name Here" size="medium" />
      </Segment>
      { !tournament ? (
        <Message>
          <Message.Header>
            No Competition Selected
          </Message.Header>
          <p>
            Please select and existing or create a new Competition on the left.
          </p>
        </Message>
      ) : (
        games.map(game => (
          <Game key={game.id} {...game} />
        ))
      )}
    </Segment.Group>
  </Container>
);

Play.propTypes = {
  tournament: PropTypes.bool,
};

export default Play;
