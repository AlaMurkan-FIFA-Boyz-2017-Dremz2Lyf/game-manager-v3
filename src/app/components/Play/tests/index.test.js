import { shallow } from 'enzyme';
import React from 'react';

import Play from '../index';

describe('<Play />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Play />);
  });

  it('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
