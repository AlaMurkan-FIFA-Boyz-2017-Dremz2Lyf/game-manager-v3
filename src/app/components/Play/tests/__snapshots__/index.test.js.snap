// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Play /> should match snapshot 1`] = `"<div class=\\"ui container\\"><div class=\\"ui segments\\"><div class=\\"ui clearing segment\\"><div class=\\"ui medium header\\"><i aria-hidden=\\"true\\" class=\\"soccer icon\\"></i><div class=\\"content\\">Tournament Name Here</div></div></div><div class=\\"ui message\\"><div class=\\"header\\">No Competition Selected</div><p>Please select and existing or create a new Competition on the left.</p></div></div></div>"`;
